py-rse/errors.Rmd

```diff
@@ -434,45 +481,48 @@ without editing the program's source code.
 We would probably wind up with code like this:
 
 ```python
-DEBUG_LEVEL = 2 # or some other number
+LOG_LEVEL = 2  # Or some other number
 
 def complex_function():
     # ...lots of complicated code...
-    if (DEBUG_LEVEL > 0):
+    if (LOG_LEVEL > 0):
         print('Everything seems to be going well')
     # ...more complicated code...
-    if (DEBUG_LEVEL > 1):
+    if (LOG_LEVEL > 1):
         print('Current alpha value {}'.format(alpha))
 ```
 
 Here,
-`DEBUG_LEVEL` acts as a threshold:
+`LOG_LEVEL` acts as a threshold:
 any debugging output at a lower level isn't printed.
 
 Logging frameworks combine the `if` and the `print` statements,
-and define standard names for the levels.
-In order of increasing severity, the levels are:
+and define standard names for the logging levels.
+In order of decreasing severity, the levels are:
 
--   `DEBUG`: very detailed information used for localizing errors.
--   `INFO`: confirmation that things are working as expected.
--   `WARNING`: something unexpected happened, but the program will keep going.
--   `ERROR`: something has gone badly wrong, but the program hasn't hurt anything.
 -   `CRITICAL`: potential loss of data, security breach, etc.
+-   `ERROR`: something has gone badly wrong, but the program hasn't hurt anything.
+-   `WARNING`: something unexpected happened, but the program will keep going.
+-   `INFO`: confirmation that things are working as expected.
+-   `DEBUG`: very detailed information used for localizing errors.
 
 Each of these has a corresponding function:
 we can use `logging.debug`, `logging.info`, etc. to write messages at these levels.
 By default,
 only `WARNING` and above are displayed;
-messages appear on [standard error][standard-error] so that the flow of data in pipes isn't affected.
+messages appear on [standard error][standard-error]
+so that the flow of data in pipes isn't affected.
 The source of the message appears as well:
-by default, the source is called "root" for reasons that will become clear later.
+by default,
+the source is called "root" for reasons that will become clear later.
```

Comment:
should "root" be in code font?

Reply:
Also, can you please add a forward ref to the explanation?

Reply:
I didn't see an explanation (and not sure if one is needed for this), so I will remove the reference.